src/bedriftsmeny/byggOrganisasjonsTre.ts

```diff
@@ -1,71 +1,38 @@
 import { JuridiskEnhetMedUnderEnheterArray, Organisasjon } from './organisasjon';
 import { hentAlleJuridiskeEnheter } from './hentAlleJuridiskeEnheter';
 
-const gyldigeUnderenheter = ['BEDR', 'AAFY'];
+const erHovedenhet = (organisasjon: Organisasjon): boolean =>
+    !!organisasjon.OrganizationNumber &&
+    (organisasjon.Type === 'Enterprise' || organisasjon.OrganizationForm === 'FLI');
 
-const erGyldigUnderenhet = (form: string) => {
-    return gyldigeUnderenheter.includes(form);
-};
+const erUnderenhet = (organisasjon: Organisasjon): boolean =>
+    !!organisasjon.OrganizationNumber
+    && ['BEDR', 'AAFY'].includes(organisasjon.OrganizationForm);
 
 export async function byggOrganisasjonstre(
     organisasjoner: Organisasjon[]
 ): Promise<JuridiskEnhetMedUnderEnheterArray[]> {
-    const juridiskeEnheter = organisasjoner.filter(function(organisasjon: Organisasjon) {
-        return organisasjon.Type === 'Enterprise' || organisasjon.OrganizationForm === 'FLI';
-    });
+    organisasjoner = organisasjoner.sort((a, b) => a.Name.localeCompare(b.Name))
```

Comment:
sort manipulerer in place, så trenger ikke assign her. Evt en tryggere approach kan være å lage et nytt array ala `const sortert = [...organisasjoner].sort((a, b) => a.Name.localeCompare(b.Name))`